packages/block-editor/src/components/block-preview/index.js

```diff
@@ -20,11 +21,41 @@ import { BlockListItems } from '../block-list';
 
 export function BlockPreview( {
 	blocks,
-	__experimentalPadding = 0,
 	viewportWidth = 1200,
+	minHeight,
+	additionalStyles = [],
+	// Deprecated props:
 	__experimentalMinHeight,
-	__experimentalStyles = [],
+	__experimentalStyles,
+	__experimentalPadding,
 } ) {
+	if ( __experimentalMinHeight ) {
+		minHeight = __experimentalMinHeight;
+		deprecated( 'The __experimentalMinHeight prop', {
+			since: '6.2',
+			alternative: 'minHeight',
+		} );
+	}
+	if ( __experimentalStyles ) {
+		additionalStyles = __experimentalStyles;
+		deprecated( 'The __experimentalStyles prop of BlockPreview', {
```

Comment:
This hasn't landed in core yet, so we can skip the `deprecated` call here.

Reply:
Yeah ok, I'll remove the migration for this altogether. I confirmed on wpdirectory.net that nothing seems to be using `__experimentalStyles`.